### PR TITLE
#352 Created integration test

### PR DESCRIPTION
**Pull request by alevohin**
issue #352. Created integration test

### CODE REVIEW

**Comment by davvd** [comment]
I'll find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by alevohin** [comment]
@davvd PR is stable, please find a reviewer.

**Comment by alevohin** [comment]
@davvd PR is stable, please find a reviewer.

**Comment by davvd** [comment]
@krzyk this pull request is for you, please review

File: qulice-maven-plugin/src/it/findbugs-exclude/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <plugins>
+            <plugin>
+                <groupId>com.qulice</groupId>
+                <artifactId>qulice-maven-plugin</artifactId>
+                <version>@project.version@</version>
+                <configuration>
+                    <license>file:./LICENSE.txt</license>
+                    <excludes>
+                        <!-- @todo #352 Need to add multiline check after issue #350-->
+                        <!-- resolved. Now only one class can be excluded -->
```
Comment (krzyk) [inline qulice-maven-plugin/src/it/findbugs-exclude/pom.xml]: add a reminder that `example-exclude.apt.vm` has to be updated also

File: qulice-maven-plugin/src/site/apt/example-exclude.apt.vm [hunk 2]
```diff
@@ -70,3 +71,13 @@ Exclude
   folders;
   2) <<<codenarc:**/src/ex1/Main.groovy,**/src/ex2/Main2.groovy>>> excludes
   <<<Main.groovy>>> and <<<Main2.groovy>>> in appropriate packages.
+
+  Exclude for Findbug is a bit different, as expression must be exact class name
```
Comment (krzyk) [inline qulice-maven-plugin/src/site/apt/example-exclude.apt.vm]: slightly reworded that: `..as expression has to be either an exact class name or start with the..`

**Comment by krzyk** [comment]
@alevohin two minor comments above

**Comment by alevohin** [comment]
@krzyk thanks! Please check one more time.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by krzyk** [comment]
@alevohin there is error from PDD:
> Space expected in line #49 in /home/r/repo/qulice-maven-plugin/src/it/findbugs-exclude/pom.xml (PDD::Error)

**Comment by krzyk** [comment]
@alevohin BTW. you can try to install https://github.com/teamed/pdd and try it before the build and then run it with `pdd --source=$(pwd) --verbose --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java` (but it's always goot to check rultor log for the exact command)

**Comment by alevohin** [comment]
@krzyk thanks. Please check this PR one more time.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge it

**Comment by davvd** [comment]
@krzyk Much appreciated! **18 mins** was added to your account, payment ID `50344410`

+18 added to your rating, current score is: [+1996](http://www.netbout.com/b/35035?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**